tests/cupyx_tests/fallback_mode_tests/test_fallback.py

```diff
@@ -235,17 +235,18 @@ def test_fallback_methods_array_external_out(self, xp):
 
 
 @testing.parameterize(
-    {'object': fallback_mode.numpy.ndarray},
-    {'object': fallback_mode.numpy.ndarray.__add__},
-    {'object': fallback_mode.numpy.vectorize},
-    {'object': fallback_mode.numpy.linalg.eig},
+    {'object': ['ndarray']},
+    {'object': ['ndarray', '__add__']},
+    {'object': ['vectorize']},
+    {'object': ['linalg', 'eig']},
 )
 @testing.gpu
 class TestDocs(unittest.TestCase):
 
     @numpy_fallback_equal()
     def test_docs(self, xp):
-        return getattr(self.object, '__doc__')
+        obj = functools.reduce(getattr, self.object, xp)
+        return getattr(obj, '__doc__')
```

Comment:
How about using `operator.attrgetter`?
```
>>> operator.attrgetter('linalg.eig')
operator.attrgetter('linalg.eig')
>>> operator.attrgetter('linalg.eig')(numpy).__doc__[:100]
'\n    Compute the eigenvalues and right eigenvectors of a square array.\n\n    Parameters\n    ---------'
```

Reply:
That's neat! I learned a new thing today 😄